templates/gmetad.conf.erb

```diff
@@ -102,12 +102,16 @@ gridname "<%= scope.lookupvar('::ganglia::gmetad::gridname') %>"
 # is always trusted.
 # default: There is no default value
 # trusted_hosts 127.0.0.1 169.229.50.165 my.gmetad.org
+trusted_hosts <%= @trusted_hosts.join(' ') %>
```

Comment:
This causes failures under the future parser  (which are allowed to fail in the travis matrix due to a change in erb `:undef` handing) -- could you add a condition check for nil and check the output of the future parser runs?